### PR TITLE
Fixed the link when the audited comic is not available [#1199]

### LINKED ISSUE #1199: When a metadata audit log entry references a missing comic, it should not show a comic link.
As a system, when a metadata audit log entry references a comic that's no longer in the library, then it should not show a link. Instead it should instead show text saying the comic is no longer available.

### PR DESCRIPTION
**Pull request by mcpierce**
Also fixed how Jackson encodes the ID for objects, so that it
uses the actual id property rather than generating a new UUID.

Closes #1199 

## Status
READY

## Does this PR contain migrations?
NO

## Before You Submit Your PR:
- [ ] Have you announced your PR on the comixed-dev mailing list?
- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactors existing code (code changes for efficiency or maintainability)
- [ ] Security fix (be sure to include the CVE in the commit message) 
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: comixed-webui/src/assets/i18n/es/comic-metadata.json [hunk 1]
```diff
@@ -97,11 +100,11 @@
...
-      "cancel-scraping": "Deja de raspar cómics.",
-      "reject-scraping-issue": "Rechazar este problema.",
-      "select-scraping-issue": "Seleccione este problema.",
-      "show-comic-details": "Mostrar detalles",
-      "skip-cache": "Raspado será {enabled, select, true{skip} other{use}} la caché de raspado.",
+      "cancel-scraping": "Stop scraping comics.",
+      "reject-scraping-issue": "Reject this issue.",
+      "select-scraping-issue": "Select this issue.",
+      "show-comic-details": "Show Details",
+      "skip-cache": "Scraping will {enabled, select, true{skip} other{use}} the scraping cache.",
```
Comment (BRUCELLA2) [inline comixed-webui/src/assets/i18n/es/comic-metadata.json]: See PR #1198 comments

File: comixed-webui/src/assets/i18n/fr/comic-metadata.json [hunk 2]
```diff
@@ -97,12 +100,12 @@
...
-      "select-scraping-issue": "Sélectionner ce numéro.",
-      "show-comic-details": "Afficher les détails",
-      "skip-cache": "La récupération des détails {enabled, select, true{évitera} other{utilisera}} le cache de récupération.",
-      "use-metadata-source-id": "Récupération des métadonnées en utilisant l'identifiant ComicVine."
+      "cancel-scraping": "Stop scraping comics.",
+      "reject-scraping-issue": "Reject this issue.",
+      "select-scraping-issue": "Select this issue.",
+      "show-comic-details": "Show Details",
+      "skip-cache": "Scraping will {enabled, select, true{skip} other{use}} the scraping cache.",
+      "use-metadata-source-id": "Scrape using the ComicVine id."
```
Comment (BRUCELLA2) [inline comixed-webui/src/assets/i18n/fr/comic-metadata.json]: See PR #1198 comments

**Comment by BRUCELLA2** [comment]
Need to revert the translation modification for fr and es and it's ok

**Comment by mcpierce** [comment]
> Need to revert the translation modification for fr and es and it's ok

Done.

**Review by BRUCELLA2 — APPROVED** [review]

**Pull request merged**